wallet/node/Main.hs

```diff
@@ -49,10 +51,18 @@ actionWithWallet sscParams nodeParams wArgs@WalletArgs {..} =
   where
     convPlugins = (, mempty) . map (\act -> ActionSpec $ \__vI __sA -> act)
     plugins :: HasConfigurations => ([WorkerSpec WalletWebMode], OutSpecs)
-    plugins = convPlugins (pluginsGT wArgs) <> walletProd wArgs
+    plugins = mconcat [ convPlugins (pluginsGT wArgs)
+                      , walletProd wArgs
+                      , acidCleanupWorker wArgs ]
+
+acidCleanupWorker :: HasConfigurations => WalletArgs -> ([WorkerSpec WalletWebMode], OutSpecs)
+acidCleanupWorker WalletArgs{..} =
+    first one $ worker mempty $ const $
+    modifyLoggerName (const "acidcleanup") $
+    cleanupAcidStatePeriodically walletAcidInterval
 
 walletProd :: HasConfigurations => WalletArgs -> ([WorkerSpec WalletWebMode], OutSpecs)
-walletProd WalletArgs {..} = first pure $ worker walletServerOuts $ \sendActions ->
+walletProd WalletArgs {..} = first one $ worker walletServerOuts $ \sendActions ->
```

Comment:
What was this change for? Is it related?

Reply:
Not related, i just got confused by using `pure` as a single element list constructor for half a second and decided to clarify :)